vue/sbc-common-components/src/plugins/vuetify.ts

```diff
@@ -54,8 +54,10 @@ export default new Vuetify({
         },
         bcgovgold: {
           base: '#fcba19'
-        }
+        },
+        appNotificationOrange: '#FCBA19'
```

Comment:
Yes, this is the way to declare new colour variables that you can use in Vuetify.

But are you using this anywhere?

Reply:
I didn't use this. I have it defined in theme.scss . Very confused about scss variables. Some are defined in vuetify.ts, most are defined in scss. Is there any standard for this?

Reply:
In theme.scss it's an actual _CSS variable_ that you can use in other CSS.

In this file are the _Vuetify colour variables_ used by Vuetify components.

For example, instead of doing this:
`<v-btn color="#FCBA19">`

You can do this:
`<v-btn color="appNotificationOrange">`

In the example above, "color" is not a style/CSS, it's a Vuetify component prop.

As for a standard, I like to have no hex colours in the components -- named colours (or variables) only. Therefore for some elements we need to use CSS variables, and for some Vuetify components we need to use Vuetify colour variables. I try to name them the same, and also usually leave a comment on them, like this: [appNotificationOrange](https://github.com/bcgov/business-filings-ui/blob/42e669aff96183e900ceed1432dff0c2687edeee/src/main.ts#L74)

Reply:
I tried to do ```<v-app-bar color="appNotificationOrange">```. But it doesn't work and so I put it in css 

```
.v-app-bar.v-toolbar.v-sheet {
  background-color: $app-notification-orange !important;
}
```

Reply:
Sure.